doc/compiler/jitserver/Usage.md

```diff
@@ -20,14 +20,16 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-JITServer adds an additional two *personas* to the `java` executable: client mode and server mode. There are two new command line options that can be used to select a persona: `-XX:StartAsJITServer` and `-XX:+UseJITServer`.
+JITServer adds an additional two *personas* to the JVM: client mode and server mode. To start the JVM in server mode, run the `jitserver` launcher present under `bin` directory (along side `java` launcher). To start the JVM in client mode, there is a new command line option: `-XX:+UseJITServer`.
 
-In server mode, the JVM will halt after startup and begin listening for compilation requests from clients. No Java application is given to the server on the command line.
+In server mode, the JVM will halt after startup and begin listening for compilation requests from clients. No Java application is given to the `jitserver` launcher on the command line. Any option accepted by JVM can also be passed to the `jitserver` launcher.
 
 ```
-$ java -XX:StartAsJITServer
+$ jitserver
```

Comment:
Could we add a path to the command to show where the launcher can be found to make it more explicit for people who are not familiar with the build directory structures? 
Such as for Java 8: 
`$~/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/bin/jitserver`

Reply:
There is a comment above that the launcher is present in the `bin` directory alongside the `java` launcher. In my mind this is sufficient to locate it